### ISSUE TITLE
[2106.10270] How to train your ViT? Data, Augmentation, and Regularization in Vision Transformers

### ISSUE THREAD

**Issue by riron1206** [riron1206/read_paper#94]
## 概要
https://arxiv.org/abs/2106.10270
どうしたら ViT の学習うまくいくか調べた論文
以下のことしてる
- ViT を事前学習する際の正則化，Data Augmentation，モデルサイズ，学習データサイズの相互作用について調査
- ViT の転移学習についても調査

**ViT の事前学習は適切な正則化（Dropout, StochasticDepth, Weight decay）とData Augmentation（RandAugment, Mixup）=AugReg を入れると、データセットのサイズを1桁増やすのと同様の効果が得られることがわかった**

**AugReg は転移学習では効果ないので注意！！！AugReg は事前学習でのみ有効（Figure 1 より）**

正則化はでかいモデル（ViT Large とか）に効果があり、長時間のトレーニングを行った場合にのみ効果がある（Figure 7 より）
精度の向上（緑，正の数）と低下（赤，負の数）を示してる
![image](https://user-images.githubusercontent.com/22429441/122849226-12b5ae00-d346-11eb-88b0-2a7140b6ef5e.png)

下流のデータが事前学習に使用したデータと関連性が低いと思われる場合でも、ViT の転移学習は効果ある
- 小さいデータセットの精度は、事前学習無しのViT < 転移学習したViT になった
- CNN と同様に長く学習したでかい事前学習モデルで転移学習した方が精度上がりやすい（Figure 3, Table 5 より） 
![image](https://user-images.githubusercontent.com/22429441/122849346-4a245a80-d346-11eb-8cbf-c869687d88e0.png)

![image](https://user-images.githubusercontent.com/22429441/122849279-2a8d3200-d346-11eb-943e-30410b4cc63f.png)

## コメント
参考サイト
